### ISSUE TITLE
Feature Request: Swipe Gesture to toggle between Edit & Preview Modes

### ISSUE THREAD

**Issue by ASNosal** [etesync/etesync-notes#39]
When editing a note, sometimes you just want to quickly toggle into Preview mode and back, and an even faster more convenient way to do this would be implement a  swipe gesture to toggle between Edit & Preview modes.

This is a feature that I use in Markor constantly, so I'd love to see it make it's way into Etesync Notes.

**Comment by tasn** [comment]
I just tried it in markor and I couldn't trigger it. How do I do it?

**Comment by ASNosal** [comment]
It's just a swipe from the center-ish of the screen directly left or right towards one of the sides when editing or previewing a markdown note.

**Comment by tasn** [comment]
In Markor? Are you sure? Doesn't work here.

**Comment by ASNosal** [comment]
Yes - I tried looking through the settings to see if there was a setting I maybe enabled for it, but I can't seem to find anything. Here's a real quick screen capture of the action:

https://drive.google.com/file/d/165f3biAC8Geyl8Qi4jknnRUVKAPqgV_5/view?usp=sharing

**Comment by tasn** [comment]
Weird, can't get it to happen here! Though I can guess what may be the difference:
I'm using markor from f-droid, maybe the Google Play version uses some proprietary APIs for gesture recognition.

**Comment by ASNosal** [comment]
I'm actually using the F-Droid version myself as well, so I don't understand why it's not working for you!

It looks like it was added in V1.7 with an option added in V1.8: [https://github.com/gsantner/markor/blob/master/CHANGELOG.md](https://github.com/gsantner/markor/blob/master/CHANGELOG.md) - but I don't seem to see that option anywhere....
